src/classlibnative/bcltype/bignum.cpp

```diff
@@ -0,0 +1,599 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// File: bignum.cpp
+//
+
+//
+
+#include "bignum.h"
+#include <intrin.h>
+
+constexpr UINT32 BigNum::m_power10UInt32Table[UINT32POWER10NUM];
+BigNum BigNum::m_power10BigNumTable[BIGPOWER10NUM];
```

Comment:
It needs to be initialized in https://github.com/mazong1123/coreclr/blob/9b39ea6c0e28846dfdf327530d1ea84210bff0c7/src/classlibnative/bcltype/bignum.h#L73 so it probably cannot be `const` or `constexpr`